test/controllers/secure.register.filter.controller.spec.ts

```diff
@@ -43,6 +45,9 @@ const mockSetExtraData = setExtraData as jest.Mock;
 const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
 mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
 
+const mockRemoveJourneyMiddleware = removeJourneyMiddleware as jest.Mock;
```

Comment:
Tested locally. Seems to work, but quite a few tests fail.

Would need to add this to each failing test or is there some way to define a general mock implementation across tests @markpit ?

Reply:
yes, you can create the mock in a separate file and import it

Reply:
eg https://github.com/companieshouse/confirmation-statement-web/blob/main/test/controllers/create.transaction.controller.unit.ts#L5

Reply:
Thanks. Looks simple enough. But we would then need to add the import statement to every (failing) test file, yes?

Reply:
yes, you'd still need to import it